src/components/search_results.js

```diff
@@ -17,13 +16,12 @@ export default class SearchResults extends React.Component {
   // componentDidMount() {}
 
   render() {
-    const array = _.range(20)
     let image = 'https://via.placeholder.com/200'
     return(
       <div className="font-playfair font-normal text-3xl text-center mx-56">
         <h2 className="mb-16">Search results for {this.state.query}</h2>
         <div className="grid grid-cols-5 gap-4 justify-items-stretch">
-          {array.map((i) => {
+          {Array.from({length: 20}, (v, i) => {
```

Comment:
Should we be limiting this on the backend with pagination possibly? Seems wasteful for us to send a full array only to have only the first 20 displayed. That could be a refactor though, doesn't need to be in the MVP

Reply:
Yes... I did not do any pagination for the Cocktail Search, so if it's a long list, then it will take a while for the user to scroll down the page. Having said that, I haven't found any that long of a list when taking a look at the Cocktail DB api site... Still doesn't hurt to refactor at some point and limit per page.